docs/cep/query-guide/functions/geospatial/proximity.md

```diff
@@ -23,8 +23,29 @@ Determines if the specified object or location is within the radius of another o
 ## Example 1
 
 ```sql
-@info(name = 'query1')
 geo:proximity(1, 0, 0, 110574.61087757687)
 ```
 
-This example returns `true` because the coordinates `(1, 0)` are within the given radius of `110574.61087757687` from the reference point `(0, 0)`.
+The `geo:proximity()` function is used to determine if the point (1, 0) lies within the radius of 110574.61087757687 units from the reference point (0, 0). It returns `true`, indicating that the point is indeed within the given radius from the reference location.
+
+## Example 2
+
+```sql
+geo:proximity(2, 0, 0, 110574.61087757687)
+```
+
+In this case, the `geo:proximity()` function checks if the point (2, 0) lies within the radius of 110574.61087757687 units from the reference point (0, 0). The function returns `false` as the point is not within the given radius from the reference point.
+
+## Example 3
+
+```sql
```

Comment:
And this one

Reply:
@StoyanTc / @diana-macrometa -  It seems the last parameter radius double for geo:proximity stream processor is not accepting dynamic values... The SW only validates if we give a hardcoded double value to the radius parameter..
Is this expected or a bug??

2. @StoyanTc Even if I hardcode radius  to say 0.5, the SW validates but it doesn't work, I don't receive anything in OutputGeoStream, can you check there is some bug or the example I used is wrong?? Below are the details...

 Input given to InputGeoStream is `{"id":"person234", "longitude": 9.876657, "latitude": 80.897648, "geoJsonFence":"{'type':'Polygon','coordinates':[[[0, 0],[0, 1],[1, 1],[1, 0],[0, 0]]]}"}`

SW used is:
`CREATE STREAM InputGeoStream (id string, longitude double, latitude double, geoJsonFence string);
CREATE SINK STREAM OutputGeoStream (id string, proximityResult bool);

@info(name = 'proximityCheck')
INSERT INTO OutputGeoStream
SELECT id, inCloseProximity AS proximityResult
FROM InputGeoStream#geo:proximity(id, longitude, latitude, geoJsonFence, 0.5);`

Reply:
@StoyanTc, please take a look at this one.